internal/configs/virtualserver_test.go

```diff
@@ -895,6 +925,33 @@ func TestGenerateProxyPassProtocol(t *testing.T) {
 	}
 }
 
+func TestGenerateString(t *testing.T) {
+	tests := []struct {
+		inputS   string
+		expected string
+	}{
+		{
```

Comment:
this comment wasn't fully addresses -- https://github.com/nginxinc/kubernetes-ingress/pull/628#discussion_r306830297